src/simulator/base.py

```diff
@@ -0,0 +1,39 @@
+from collections import Counter
+from typing import List
+
+import typing
+
+from src.simulator.consts import GOLD_SCORE, NUM_OF_DICE, SCORE_DICT
+from src.simulator.dice import Dice, DiceValues
+
+
+class Base:
+    """This class implements the logic of a regular turn. Special cases such as captain, angel, skull and etc should
+    inherit from this class and update its functions. """
+    @staticmethod
+    def dice_count(dice_list: List[Dice]) -> typing.Counter[Dice]:
```

Comment:
Which python version are using? Starting from python3.9 you can use list[Dice] (builtin list instead of typing.List).
Anyway, you have both from typing import List and import typing, so keep the same convention (either typing.List or import Counter)

Reply:
I will upgrade my python version. thanks for the comment